language_detection/python/train.py

```diff
@@ -11,15 +11,15 @@
 
 file_list = glob.glob('../dat/*_train.txt')
 
-fds = map(lambda x: [x.replace("_train.txt", ""), open(x, "r")], file_list)
+fds = [[x.replace("_train.txt", ""), open(x, "r")] for x in file_list]
```

Comment:
List comprehension syntax (``[ ... for x in ... ]``) won't work with Python 2.6.